ssv/spectest/tests/runner/full_happy_flow.go

```diff
@@ -205,8 +231,34 @@ func FullHappyFlow() tests.SpecTest {
 		}
 	}
 
+	// proposerBlindedReceivingNormalBlockV creates a test specification for versioned blinded proposer receiving a normal block.
```

Comment:
I would separate it into a separate test, testing specific cases in which some of the cluster members propose blinded blocks and others don't

Reply:
I took it out of the happy flow test and put it in a file of its own in runner/proposer. What do you think?